dbtwiz/commands/backfill.py

```diff
@@ -0,0 +1,173 @@
+from datetime import date
```

Comment:
What are your thoughts on having a docstring at the top of each file, describing in general terms what the script does?

Reply:
It's a good rule, perhaps create an issue for it? It would be way too much for this PR which is already too big.

Reply:
Added an issue: https://github.com/amedia/dbtwiz/issues/28